### PR TITLE
Add doc about Upgrade/Downgrade ProActive using Docker

### PR DESCRIPTION
**Pull request by mboussaa**

### CODE REVIEW

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 1]
```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
...
+==== Upgrade/Downgrade ProActive using Docker Compose
+
+The ProActive version could be easily upgraded using Docker Compose. To do so, we support automatic upgrade, downgrade, and backup of ProActive Server installations in Docker.
+Docker Compose configuration relies on Docker container volumes to host ProActive versions. The `proactive-default` volume hosts the current and default installation. The `proactive-previous` volume hosts the previous installation after upgrade.
+Finally, the `proactive-backup` volume hosts all previous installtion after multiple upgrades.
+In the following, we are going to explain each procedure.
+
+===== ProActive Upgrade
+In order to upgrade the ProActive Server to a newer version, you have to edit the `.env` config file to provide a newer version. The current ProActive version is specified using the `PROACTIVE_VERSION` option.
+For example, you can replace `PROACTIVE_VERSION=12.0.0` by `PROACTIVE_VERSION=12.0.8` to get a maintenance release of ProActive.
```
Comment (aminelouati) [inline src/docs/admin/ProActiveAdminGuide.adoc:546]: Does this upgrade cover major and minor releases ? If yes could you add a mention about that ?
What happen if someone provide an anterior version like 11.0.0 ?

**Review by aminelouati — APPROVED** [review]
minor comments

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 1 above)
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:546]: ok

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== Upgrade/Downgrade ProActive using Docker Compose
```
[the remark this answers is not in the document]
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:537]: ok

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
 
 Your ProActive Scheduler is now running as a set of container services!
 
+==== Upgrade/Downgrade/Backup ProActive using Docker Compose
+
+The ProActive version could be easily upgraded using Docker Compose. To do so, we support automatic upgrade, downgrade, and backup of ProActive Server installations in Docker.
```
Comment (nebilbenmabrouk) [inline src/docs/admin/ProActiveAdminGuide.adoc:539]: I suggest `ProActive can be easily upgraded using Docker Compose.`

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 4]
```diff
@@ -534,6 +534,29 @@ The user can access any of the four portals using the default credentials: `admi
...
+Docker Compose configuration relies on Docker container volumes to host ProActive versions. The `proactive-default` volume hosts the current and default installation. The `proactive-previous` volume hosts the previous installation after upgrade.
+Finally, the `proactive-backup` volume hosts all previous installtion after multiple upgrades.
+In the following, we are going to explain each procedure.
+
+===== ProActive Upgrade
+In order to upgrade the ProActive Server to a newer version, you have to edit the `.env` config file to provide a newer version. The current ProActive version is specified using the `PROACTIVE_VERSION` option.
```
Comment (nebilbenmabrouk) [inline src/docs/admin/ProActiveAdminGuide.adoc:545]: In order to upgrade ProActive server ..

**Review by nebilbenmabrouk — APPROVED** [review]

File: src/docs/admin/ProActiveAdminGuide.adoc (hunk 3 above)
Comment (mboussaa) [reply src/docs/admin/ProActiveAdminGuide.adoc:539]: ok

**Comment by mboussaa** [comment]
Thx @nebilbenmabrouk @aminelouati for the reviews

**Pull request merged**